integrationTests/chainSimulator/common.go

```diff
@@ -58,8 +58,8 @@ type Account struct {
 	Nonce  uint64
 }
 
-// GetSysAccBytesAddress will return the system account bytes address
-func GetSysAccBytesAddress(t *testing.T, nodeHandler process.NodeHandler) []byte {
+// GetSysContactDeployAddressBytes will return the system contract deploy address
+func GetSysContactDeployAddressBytes(t *testing.T, nodeHandler process.NodeHandler) []byte {
```

Comment:
```suggestion
func GetSysAccAddressBytes(t *testing.T, nodeHandler process.NodeHandler) []byte {
```

Reply:
That contract is a system contract for deploy, that's why I renamed the function

Reply:
Everyone calls it sys acc, name is just too long

Reply:
W/e. can stay as it is